build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ $# -ne 2 ]
+then
+    echo "Syntax: build.sh <app_name> <tf_workspace>"
+    sleep 30
+    exit
+fi
+cd infrastructure
+terraform init -backend-config "bucket=$1-terraform-state"
+terraform workspace new $2 || terraform workspace select $2
+terraform apply -auto-approve -input=true

```

Comment:
I'm not 100% sure we should have auto approve on by default, but it is easier I guess

Reply:
for project setup it works given that what's in github should work, during development we wouldn't do that - I'll add a task for documentation steps during development